.dockerignore

```diff
@@ -0,0 +1,57 @@
+**/node_modules
+public/webpack
+.git
+.bundle
+~
+db/*.sqlite3*
+log/*.log
+log/*.log.age
+tmp/
+jenkins/reports/*
+test/reports/*
+db/schema.rb
+config/settings.yaml
+config/ignored_environments.yml
+config/settings.plugins.d
+config/database.yml
+config/initializers/local_secret_token.rb
+config/initializers/encryption_key.rb
+*.sw?
+.idea
+*.pyc
+public/apipie-cache
+public/assets
+.DS_Store
+foreman_client
+doc/
+.rbenv*
+.rvmrc*
+.ruby-version*
+.ruby-gemset*
+locale/*.mo
+locale/*/*.edit.po
+locale/*/*.po.time_stamp
+locale/*/*.pox
+locale/*/LC_MESSAGES
+coverage/
+tags
+_build
+zeus.json
+custom_plan.rb
+pkg/
+config/hooks
+*.csv
+vendor/ruby
+.env*
+package-lock.json
+npm-debug.log
+.vscode
+.yo-rc.json
+out*
+.byebug_history
+.gitignore
+.dockerignore
+.travis.yml
+kube
```

Comment:
??

Reply:
I'm sorry, that's a leftover from from another work (if you are interested you can see it at https://github.com/ohadlevy/foreman/tree/dockerfile/kube) will remove..